reana_job_controller/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.7.2"
+__version__ = "0.7.3"
```

Comment:
Please amend `CHANGES.rst` describing the fix.

Reply:
Also, we should replace "(UNRELEASED)" with the release date.